core/tests/test_account/src/lib.rs

```diff
@@ -135,14 +135,18 @@ impl Account {
             value: U256::zero(),
         };
 
+        let mut nonce = 0;
```

Comment:
please add a comment here that both using txs nonce or serial id is not exactly a correct way to calculate the deployed address on era, but it is used here for easier testing (and also tell which way is the correct one)

Reply:
What is the correct one to calculate deployed address of era? so I can write a precise comment

Reply:
by using deployment nonce, which is stored separately from the usual nonce and is incrementally only if there are contract deployments